packages/xstate-solid/test/deepClone.test.ts

```diff
@@ -0,0 +1,68 @@
+import { deepClone } from '../src/deepClone';
+
+describe('Deep Clone', () => {
```

Comment:
are all of those cases also tested indirectly in the other tests?

Reply:
Some of them are, the circular reference test is not. I added these tests as a base to validate that future bugs aren't coming from the deepClone function itself. They more of a nice to have, so I don't see a problem with adapting the circular reference test into one of the other hooks

Reply:
i think that it would be good to add a test at a higher level, in addition to this one here